NitroxClient/Debuggers/Drawer/Unity/RigidbodyDrawer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using UnityEngine;
+
+namespace NitroxClient.Debuggers.Drawer.Unity
+{
+    /// <summary>
+    /// Draws a Rigidbody component on the gameobjects in the <see cref="SceneDebugger"/>
+    /// </summary>
+    public class RigidbodyDrawer : IDrawer
+    {
+        private const float LABEL_WIDTH = 120;
+        private const float VALUE_MAX_WIDTH = 405;
+
+        public Type[] ApplicableTypes => new[] { typeof(Rigidbody) };
+
+        public void Draw(object target)
+        {
+            Rigidbody rb = (Rigidbody)target;
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Velocity", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                VectorDrawer.DrawVector3(rb.velocity, VALUE_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Angular Velocity", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                VectorDrawer.DrawVector3(rb.velocity, VALUE_MAX_WIDTH);
+            }
```

Comment:
Normally we only expose all fields (if possible) that are also shown in the Unity Editor (see [here](https://docs.unity3d.com/2019.4/Documentation/Manual/class-Rigidbody.html) for rigidbody). So if you have the time it would be great if you could implement the rest of the fields. And as velocity and angular velocity are not part of this I would move them to the end.

Reply:
Added all fields except freeze rotation by xyz and freeze rotation xyz. For some reason those are not part of the Rigidbody API, but given that they can be set through the transform I don't view this as a problem.